ivy/functional/ivy/nest.py

```diff
@@ -1055,6 +1055,65 @@ def nested_map(
     ret
         x following the applicable of fn to it's nested leaves, or x itself if x is not
         nested.
+
+    Examples
+    --------
+    With :class:`Tuple` inputs:
+
+    >>> x = ([[1., 2.], [3., 4.]])
+    >>> function = lambda a : a * 2
+    >>> ivy.nested_map(x, function)
+    >>> print(x)
+    ([[2., 4.], [6., 8.]])
+
+    With :code:`Dict` input:
+
+    >>> x = {1 : [1, [2, 3]], 2: (4, 5)}
+    >>> function = lambda a : a + 1
+    >>> ivy.nested_map(x, function)
+    >>> print(x)
+    {1: [1, 2], 2: (4, 5)}
+
+    With :code:`List` inputs:
+
+    >>> x = [['a', 'b', 'c'],
+    ...      ['d', 'e', 'f'],
+    ...      ['g', ['h', 'i']]]
+    >>> function = lambda a: a + 'H'
+    >>> ivy.nested_map(x, function)
+    >>> print(x)
+    [['aH','bH','cH'],['dH','eH','fH'],['gH',['hH','iH']]]
+
+    >>> x = [[1, 2, 3, 4, 5], [23, 24, 25, 26, 27]]
+    >>> ivy.nested_map(x, include_derived = True)
```

Comment:
I guess you forgot to add a function here

Reply:
@KareemMAX Thank you for your corrections. I will add the functions where applicable. In terms of to_ignore and extra_nest_types, I am not entirely sure how to mention them, could please give me an idea ?